engine/classes/ElggPluginPackage.php

```diff
@@ -251,7 +254,7 @@ private function isNamedCorrectly() {
 			$required_id = $manifest->getID();
 			if (!empty($required_id) && ($required_id !== $this->id)) {
 				$this->errorMsg =
-					_elgg_services()->translator->translate('ElggPluginPackage:InvalidPlugin:InvalidId', [$required_id]);
+					elgg_echo('ElggPluginPackage:InvalidPlugin:InvalidId', [$required_id]);
```

Comment:
Is it possible to keep the `elgg_echo` on the same line a `$this->errorMsg`? now that the line is shorter.....